transformers-definition.json

```diff
@@ -0,0 +1,614 @@
+{
+  "name": "transformers",
+  "transformers": [
+    {
+      "name": "greenmask_boolean",
+      "supported_types": [
+        "boolean",
+        "byte_array"
+      ],
+      "parameters": [
+        {
+          "name": "generator",
+          "supported_type": "string",
+          "default": "random",
+          "dynamic": false,
+          "required": false
+        }
+      ]
+    },
+    {
+      "name": "greenmask_choice",
+      "supported_types": [
+        "string",
+        "byte_array"
+      ],
+      "parameters": [
+        {
+          "name": "choices",
+          "supported_type": "array",
+          "default": null,
+          "dynamic": false,
+          "required": true
+        },
+        {
+          "name": "generator",
+          "supported_type": "string",
+          "default": "random",
+          "dynamic": false,
+          "required": false
+        }
+      ]
+    },
+    {
+      "name": "greenmask_date",
+      "supported_types": [
+        "string",
+        "byte_array",
+        "date"
+      ],
+      "parameters": [
+        {
+          "name": "generator",
+          "supported_type": "string",
+          "default": "random",
+          "dynamic": false,
+          "required": false
+        },
+        {
+          "name": "min_value",
+          "supported_type": "string",
+          "default": null,
+          "dynamic": false,
+          "required": true
+        },
+        {
+          "name": "max_value",
+          "supported_type": "string",
+          "default": null,
+          "dynamic": false,
+          "required": true
+        }
+      ]
+    },
+    {
+      "name": "greenmask_firstname",
+      "supported_types": [
+        "string",
+        "byte_array"
+      ],
+      "parameters": [
+        {
+          "name": "generator",
+          "supported_type": "string",
+          "default": "random",
```

Comment:
Can it list all the possible values of `generator` in anther column, like for this one the possible values are 
`random`, `deterministic` but the JSON structure doesn't contain that information.

Reply:
done!